src/logs/self_monitoring/sfm.py

```diff
@@ -195,3 +206,17 @@ def _prepare_cloudwatch_metric(metric_name, value: Union[int, float, list], unit
         cw_metric["Value"] = value
 
     return cw_metric
+
+
+def _prepare_cloudwatch_metric_statistic(metric_name, unit, dimensions, minimum, maximum, sum, count) -> dict:
```

Comment:
Let's maybe make `_prepare_cloudwatch_metric` consistent with this method and take value as the last argument?

Reply:
Fixed